drivers/cc110x/include/cc110x_params.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2017 Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   drivers_cc110x
+ * @{
+ *
+ * @file
+ * @brief     cc110x board specific configuration
+ *
+ * @author    Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ */
+
+#ifndef CC110X_PARAMS_H
+#define CC110X_PARAMS_H
+
+#include "board.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Default parameters for the cc110x driver
+ *
+ * These values are based on the msbiot board
+ * @{
+ */
+#ifndef CC110X_SPI
```

Comment:
I would call this parameter `CC110X_PARAM_SPI`, same for other parameters (add `_PARAM` after `CC110X`).